ext/openssl/openssl.stub.php

```diff
@@ -103,20 +127,30 @@ function openssl_pkey_get_public($public_key): OpenSSLAsymmetricKey|false {}
  */
 function openssl_get_publickey($public_key): OpenSSLAsymmetricKey|false {}
 
-/** @deprecated */
+/**
+ * @sensitive-param $key
```

Comment:
This accept any key (public as well) and doesn't do anything so I would probably remove it.

Reply:
Dropped, also for `openssl_free_key`.